src/main/kotlin/no/nav/personbruker/dittnav/eventtestproducer/beskjed/beskjedApi.kt

```diff
@@ -8,7 +8,7 @@ import no.nav.personbruker.dittnav.eventtestproducer.config.respondForParameterT
 fun Route.beskjedApi(beskjedProducer: BeskjedProducer) {
 
     post("/produce/beskjed") {
-        respondForParameterType<ProduceBeskjedDto> { beskjedDto ->
+        this.respondForParameterType<ProduceBeskjedDto> { beskjedDto ->
```

Comment:
Klagde den på dette bare i beskjedApi og ikke for de andre typene?

Reply:
Nei, ser nå at den klager i de andre også. Man ser det bare når man åpner filene. Jeg skal oppdatere de andre også.